webpack.config.js

```diff
@@ -52,6 +52,7 @@ const loaders = [
             path.resolve(__dirname, 'node_modules/superagent'),
         ],
         loader: 'babel',
+        query: { compact: false },
```

Comment:
[Here](https://stackoverflow.com/questions/29576341/what-does-the-code-generator-has-deoptimised-the-styling-of-some-file-as-it-e) is why I added this.

Reply:
👍